puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -ex
+
+[ -e plugin ] && rm -rf plugin/
+git clone -b ${plugin_branch} ${plugin_repo} plugin
```

Comment:
Similar to another PR, I could remove the need to do this by using the MultipleSCM plugin (https://wiki.jenkins-ci.org/display/JENKINS/Multiple+SCMs+Plugin) to handle parameterized cloning. Or could do that as an enhancement if we simply wanted to replace the job as-is in Jenkins with the JJB version.